lib/puppet_x/stdlib.rb

```diff
@@ -1,4 +1,4 @@
-require 'puppet_x'
+require_relative '../puppet_x'
```

Comment:
is there maybe a rubocop rule we can use to enforce this?

Reply:
I've not seen one.. it would be cool if there was though. Even something to warn and say "maybe you should use require_relative here" would be nice.